pkg/timestream/datasource.go

```diff
@@ -170,6 +170,26 @@ func sliceFromRows(rows []*timestreamquery.Row, doubleQuotes bool) []string {
 	return res
 }
 
+// The dimensions of a row are the diffenrent columns (apart from the measure)
```

Comment:
oops typo on "different"

Also it is not related to this PR so not blocking: I wasn't sure how to highlight it specifically but there is a typo on `MesauresRequest`.